src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -450,43 +449,28 @@ public async Task<Either<ActionResult, List<ReleaseSeriesTableEntryViewModel>>>
                             result.Add(new ReleaseSeriesTableEntryViewModel
                             {
                                 Id = seriesItem.Id,
-                                IsLegacyLink = true,
                                 Description = seriesItem.LegacyLinkDescription!,
                                 LegacyLinkUrl = seriesItem.LegacyLinkUrl,
                             });
                         }
                         else
                         {
-                            // prefer getting the latest published version over an unpublished amendment
-                            var latestVersion = await _context.ReleaseVersions
-                                .LatestReleaseVersion(seriesItem.ReleaseId!.Value, publishedOnly: true)
-                                .SingleOrDefaultAsync();
+                            var latestReleaseVersion = await _context.ReleaseVersions
+                                .Include(rv => rv.Release)
+                                .LatestReleaseVersion(releaseId: seriesItem.ReleaseId!.Value)
+                                .SingleAsync();
 
```

Comment:
What happens if we grab an unpublished amendment with an updated slug, but that new slug hasn't gone live yet? If release redirects are enabled, I think this will be possible?

Previously I think we specifically grabbed the latest published release to ensure we put the correct slug in the returned view model.

Reply:
It shouldn't be possible to have release versions belonging to the same release with differing slugs after we put in a change which prevents changing the year and time identifier on amendment versions. It will be more obvious that this isn't allowed when EES-5679 makes a change to the UI. It's currently only blocked by the fact the BE no longer allows it.

Release redirects are necessary because in future release label will become part of the release slug and we will allow changing the label and add a redirect to the slug containing the old label.

I've just made another change here though because I had failing unit tests. In this change it becomes a bit clearer that the slug returned in the view model is from the release relating to the release series item, and nothing to do with the release version. The `Slug` field will be removed from `ReleaseVersion` by EES-5658.

Before

```csharp
var latestReleaseVersion = await _context.ReleaseVersions
    .Include(rv => rv.Release)
    .LatestReleaseVersion(releaseId: seriesItem.ReleaseId!.Value)
    .SingleAsync();

var release = latestReleaseVersion!.Release;

result.Add(new ReleaseSeriesTableEntryViewModel
{
    Id = seriesItem.Id,
    ReleaseId = release.Id,
    Description = release.Title,
    ReleaseSlug = release.Slug,
    IsLatest = latestReleaseVersion.Id == publication.LatestPublishedReleaseVersionId,
    IsPublished = latestReleaseVersion.Version > 1 ||
                  latestReleaseVersion is { Version: 0, Live: true }
});
```

After
```csharp
var release = await _context.Releases
    .SingleAsync(r => r.Id == seriesItem.ReleaseId);

var latestPublishedReleaseVersion = await _context.ReleaseVersions
    .LatestReleaseVersion(releaseId: seriesItem.ReleaseId!.Value, publishedOnly: true)
    .SingleOrDefaultAsync();

result.Add(new ReleaseSeriesTableEntryViewModel
{
    Id = seriesItem.Id,
    ReleaseId = release.Id,
    Description = release.Title,
    ReleaseSlug = release.Slug,
    IsLatest = latestPublishedReleaseVersion?.Id == publication.LatestPublishedReleaseVersionId,
    IsPublished = latestPublishedReleaseVersion != null
});
```

Reply:
Discussed this on Slack.

I was thinking that an unpublished amendment could change the publication slug, and that would only go live when the amendment is published, but this is incorrect. A publication slug, when changed, always goes live immediately.

I was getting confused with methodology slugs, and those can changed without going live when attached to an unpublished amendment.

TLDR the above code is fine.